static/app/views/automations/components/actionNodes.tsx

```diff
@@ -33,13 +35,13 @@ export function useActionNodeContext(): ActionNodeProps {
 }
 
 type ActionNode = {
-  action: React.ReactNode;
-  label: string;
+  action?: React.ReactNode;
+  label?: string;
 };
 
 export const actionNodesMap = new Map<ActionType, ActionNode>([
   [ActionType.AZURE_DEVOPS, {label: t('Azure DevOps'), action: <AzureDevOpsNode />}],
-  [ActionType.EMAIL, {label: t('Email'), action: <EmailNode />}],
+  [ActionType.EMAIL, {label: t('Notify on preferred channel'), action: <EmailNode />}],
```

Comment:
not for this PR but these components are a little goofy since they get rendered at load time which will only cause issues later and should be wrapped as functions or something